ctrl_z/backup.py

```diff
@@ -81,7 +95,13 @@ def full(self, db=True, skip_db=None, files=True):
         """
         logger.info("Performing full backup")
         self.rotate()
-        self.create_directories()
+        if version:
+            self.version_path = os.path.join(self.base_dir, "version")
+            self.create_directories(True)
```

Comment:
In cases where you either pass an argument or nothing, it might be nice to explicitly keyword the argument (i.e. version=True) for better readability. Now I had to go and look what the difference was between passing True vs passing nothing. Also, since version is a boolean, I would expect the default to be False rather than None.
You could also pass the version variable directly to this method which also makes it clearer.